src/cli/add.rs

```diff
@@ -129,6 +129,18 @@ pub async fn execute(args: Args) -> miette::Result<()> {
                 .collect();
 
             if let Some(git) = &dependency_config.git {
+                if !workspace
+                    .manifest()
+                    .workspace
+                    .preview()
+                    .is_enabled(KnownPreviewFeature::PixiBuild)
+                {
+                    return Err(miette::miette!(
+                        help = "Add `workspace.preview = [\"pixi-build\"]` to enable pixi build support",
```

Comment:
maybe we can add a link to the docs so user could checkout it?
I think it should be url = thingy
https://docs.rs/miette/latest/miette/trait.Diagnostic.html#method.url